Dockerfile

```diff
@@ -1,5 +1,5 @@
 # BUILDER IMAGE
-FROM python:3.9-slim-bullseye AS builder
+FROM python:3.12-slim-bullseye AS builder
```

Comment:
Is there a reason to stay on bullseye and not also bump to the default 3.12 Debian here (bookworm)?

Reply:
Good call - yes, will update